### PR TITLE
File cleanup for autotraitor.dm

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
Cleans up autotraitor.dm including:
- snake casing of variables and a function name
- removing old comments
- updating style of a for loop
- removing and replacing the use of `spawn`

## Why It's Good For The Game
Its good for contributors, so it _may_ be good for the game in the long run.

## Testing
Ran a build after changes, it compiled.

## Changelog
NPFC

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]

File: code/game/gamemodes/autotraitor/autotraitor.dm [hunk 1]
```diff
@@ -1,7 +1,3 @@
-//This is a beta game mode to test ways to implement an "infinite" traitor round in which more traitors are automatically added in as needed.
-//Automatic traitor adding is complete pending the inevitable bug fixes.  Need to add a respawn system to let dead people respawn after 30 minutes or so.
```
Comment (lewcc) [inline code/game/gamemodes/autotraitor/autotraitor.dm:2]: hmm yes, we should add a respawn system :clueless:

File: code/game/gamemodes/autotraitor/autotraitor.dm [hunk 2]
```diff
@@ -56,32 +49,27 @@
...
+	return TRUE
 
 /datum/game_mode/traitor/autotraitor/post_setup()
 	..()
-	traitorcheckloop()
+	traitor_check_loop()
 
-/datum/game_mode/traitor/autotraitor/proc/traitorcheckloop()
+/datum/game_mode/traitor/autotraitor/proc/traitor_check_loop()
 	spawn(9000)
```
Comment (lewcc) [inline code/game/gamemodes/autotraitor/autotraitor.dm:59]: Think it's possible you could turn this into a timer rather than a spawn(9000)

**Review by lewcc — COMMENTED** [review]
Wow, this code was super WIP huh.

Just one note, looks pretty good otherwise.

File: code/game/gamemodes/autotraitor/autotraitor.dm (hunk 1 above)
Comment (Burzah) [reply code/game/gamemodes/autotraitor/autotraitor.dm:2]: It has been 12 years, maybe now is the time! 🤣

File: code/game/gamemodes/autotraitor/autotraitor.dm (hunk 2 above)
Comment (Burzah) [reply code/game/gamemodes/autotraitor/autotraitor.dm]: Done!

File: code/game/gamemodes/autotraitor/autotraitor.dm [hunk 3]
```diff
@@ -56,128 +49,91 @@
...
-			//message_admins("Number of Traitors is at maximum.  Not making a new Traitor.")
-
-		traitorcheckloop()
+/datum/game_mode/traitor/autotraitor/proc/traitor_check_loop()
+	if(SSshuttle.emergency.mode >= SHUTTLE_ESCAPE)
+		return
 
+	var/player_count = 0
+	var/traitor_count = 0
+	var/possible_traitors[0]
```
Comment (DGamerL) [inline code/game/gamemodes/autotraitor/autotraitor.dm:64]:
```suggestion
	var/list/possible_traitors = list()
```
I think this is just a quite fucked way to make a list?

Comment (Burzah) [reply code/game/gamemodes/autotraitor/autotraitor.dm:64]: I missed that. Yeah, it doesn't look great.

**Review by DGamerL — APPROVED** [review]

**Pull request merged**